### PR TITLE
Copyright Headers Update (2024-05-16T11:41:01)

### PR DESCRIPTION
**Pull request by Ishirui**

### CODE REVIEW

File: include/xten/Transform/XTenMinimizeLiveTensors.h [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline include/xten/Transform/XTenMinimizeLiveTensors.h:1]: note

File: lib/Conversion/XTenNNToLinalg.cpp [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2023 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline lib/Conversion/XTenNNToLinalg.cpp:1]: note

File: lib/Conversion/XTenNNToTorch.cpp [hunk 3]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline lib/Conversion/XTenNNToTorch.cpp:1]: note

File: lib/Transform/XTenMinimizeLiveTensors.cpp [hunk 4]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline lib/Transform/XTenMinimizeLiveTensors.cpp:1]: note

File: test/Conversion/XTenNNToLinalg/Elu.mlir [hunk 5]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Conversion/XTenNNToLinalg/Elu.mlir:1]: note

File: test/Conversion/XTenNNToTorch/GroupConv2d.mlir [hunk 6]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Conversion/XTenNNToTorch/GroupConv2d.mlir:1]: note

File: test/Conversion/XTenNNToTorch/Mish.mlir [hunk 7]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Conversion/XTenNNToTorch/Mish.mlir:1]: note

File: test/Conversion/XTenNNToTorch/Round.mlir [hunk 8]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Conversion/XTenNNToTorch/Round.mlir:1]: note

File: test/Conversion/XTenNNToTorch/Sign.mlir [hunk 9]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Conversion/XTenNNToTorch/Sign.mlir:1]: note

File: test/Conversion/XTenNNToTorch/math_ops.mlir [hunk 10]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Conversion/XTenNNToTorch/math_ops.mlir:1]: note

File: test/Dialect/XTenNN/Transform/CanonicalizePass/remove_qdq_between_concats.mlir [hunk 11]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2023 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/Transform/CanonicalizePass/remove_qdq_between_concats.mlir:1]: note

File: test/Dialect/XTenNN/dequantize.mlir [hunk 12]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2023 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/dequantize.mlir:1]: note

File: test/Dialect/XTenNN/folding.mlir [hunk 13]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2023 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/folding.mlir:1]: note

File: test/Dialect/XTenNN/group_dequantize.mlir [hunk 14]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/group_dequantize.mlir:1]: note

File: test/Dialect/XTenNN/group_quantize.mlir [hunk 15]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/group_quantize.mlir:1]: note

File: test/Dialect/XTenNN/loadExternalConst.mlir [hunk 16]
```diff
@@ -1,3 +1,4 @@
+// (c) Copyright 2023 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/loadExternalConst.mlir:1]: note

File: test/Dialect/XTenNN/ops.mlir [hunk 17]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/ops.mlir:1]: note

File: test/Dialect/XTenNN/ops_invalid.mlir [hunk 18]
```diff
@@ -1,3 +1,4 @@
+// (c) Copyright 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/ops_invalid.mlir:1]: note

File: test/Dialect/XTenNN/quantize.mlir [hunk 19]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2023 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/quantize.mlir:1]: note

File: test/Dialect/XTenNN/simplify.mlir [hunk 20]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/simplify.mlir:1]: note

File: test/Dialect/XTenNN/subgraph-invalid.mlir [hunk 21]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Dialect/XTenNN/subgraph-invalid.mlir:1]: note

File: test/Transform/XTenMinimizeLiveTensors/diamonds.mlir [hunk 22]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Transform/XTenMinimizeLiveTensors/diamonds.mlir:1]: note

File: test/Transform/XTenMinimizeLiveTensors/failures.mlir [hunk 23]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2023 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Transform/XTenMinimizeLiveTensors/failures.mlir:1]: note

File: test/Transform/XTenMinimizeLiveTensors/other_subgraphs.mlir [hunk 24]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2023 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Transform/XTenMinimizeLiveTensors/other_subgraphs.mlir:1]: note

File: test/Transform/XTenMinimizeLiveTensors/print-dot-graph.mlir [hunk 25]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2023 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/Transform/XTenMinimizeLiveTensors/print-dot-graph.mlir:1]: note

File: test/aten-opt/commandline.mlir [hunk 26]
```diff
@@ -1,3 +1,5 @@
+// (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/aten-opt/commandline.mlir:1]: note

File: test/aten-opt/outputfile.mlir [hunk 27]
```diff
@@ -1,2 +1,4 @@
+// (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline test/aten-opt/outputfile.mlir:1]: note

File: utils/Dockerfile [hunk 28]
```diff
@@ -1,3 +1,5 @@
+# (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline utils/Dockerfile:1]: note

File: utils/build-llvm.sh [hunk 29]
```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env bash
+
```
Comment (laurettaSchubert) [inline utils/build-llvm.sh:2]: note

File: utils/build-torch-mlir.sh [hunk 30]
```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
+
```
Comment (laurettaSchubert) [inline utils/build-torch-mlir.sh:2]: note

File: utils/clone-llvm.sh [hunk 31]
```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env bash
+
```
Comment (laurettaSchubert) [inline utils/clone-llvm.sh:2]: note

File: utils/makellvmdocker.sh [hunk 32]
```diff
@@ -1,3 +1,4 @@
+# (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
```
Comment (laurettaSchubert) [inline utils/makellvmdocker.sh:1]: note

**Review by laurettaSchubert — COMMENTED** [review]
added 'note' on 32 files that got a new copyright header added (all others only have been updated). we need to check if the files got updated by AMD people

File: lib/Conversion/XTenNNToLinalg.cpp (hunk 2 above)
Comment (laurettaSchubert) [reply lib/Conversion/XTenNNToLinalg.cpp:1]: Liam - ok

File: lib/Transform/XTenMinimizeLiveTensors.cpp (hunk 4 above)
Comment (laurettaSchubert) [reply lib/Transform/XTenMinimizeLiveTensors.cpp:1]: Liam, Robert, Tiago - ok

File: utils/Dockerfile (hunk 28 above)
Comment (laurettaSchubert) [reply utils/Dockerfile:1]: Tiago, Robert - ok

File: utils/build-llvm.sh (hunk 29 above)
Comment (laurettaSchubert) [reply utils/build-llvm.sh:2]: Stephen - ok

File: utils/build-torch-mlir.sh (hunk 30 above)
Comment (laurettaSchubert) [reply utils/build-torch-mlir.sh:2]: Stephen - ok

File: utils/clone-llvm.sh (hunk 31 above)
Comment (laurettaSchubert) [reply utils/clone-llvm.sh:2]: Stephen, Kristof, Jeff - ok

File: utils/makellvmdocker.sh (hunk 32 above)
Comment (laurettaSchubert) [reply utils/makellvmdocker.sh:1]: Stephen - ok

File: test/Conversion/XTenNNToTorch/GroupConv2d.mlir (hunk 6 above)
Comment (laurettaSchubert) [reply test/Conversion/XTenNNToTorch/GroupConv2d.mlir:1]: Robert -ok

File: test/Conversion/XTenNNToTorch/Mish.mlir (hunk 7 above)
Comment (laurettaSchubert) [reply test/Conversion/XTenNNToTorch/Mish.mlir:1]: Tiago - ok

File: test/Conversion/XTenNNToTorch/Round.mlir (hunk 8 above)
Comment (laurettaSchubert) [reply test/Conversion/XTenNNToTorch/Round.mlir:1]: Tiago - ok

File: test/Conversion/XTenNNToTorch/Sign.mlir (hunk 9 above)
Comment (laurettaSchubert) [reply test/Conversion/XTenNNToTorch/Sign.mlir:1]: Tiago - ok

File: test/Conversion/XTenNNToTorch/math_ops.mlir (hunk 10 above)
Comment (laurettaSchubert) [reply test/Conversion/XTenNNToTorch/math_ops.mlir:1]: Tiago - ok

File: test/Dialect/XTenNN/Transform/CanonicalizePass/remove_qdq_between_concats.mlir (hunk 11 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/Transform/CanonicalizePass/remove_qdq_between_concats.mlir:1]: Tina, Ehsan, Matthias, Dominik - ok

File: test/Dialect/XTenNN/dequantize.mlir (hunk 12 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/dequantize.mlir:1]: Ehsan, Chris - ok

File: test/Dialect/XTenNN/folding.mlir (hunk 13 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/folding.mlir:1]: Tiago - ok

File: test/Dialect/XTenNN/group_dequantize.mlir (hunk 14 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/group_dequantize.mlir:1]: Liam - ok

File: test/Dialect/XTenNN/group_quantize.mlir (hunk 15 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/group_quantize.mlir:1]: Liam - ok

File: test/Dialect/XTenNN/loadExternalConst.mlir (hunk 16 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/loadExternalConst.mlir:1]: Ferdinand -ok

File: test/Dialect/XTenNN/ops.mlir (hunk 17 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/ops.mlir:1]: Liam, Max - ok

File: test/Dialect/XTenNN/ops_invalid.mlir (hunk 18 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/ops_invalid.mlir:1]: Tiago - ok

File: test/Dialect/XTenNN/quantize.mlir (hunk 19 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/quantize.mlir:1]: Ehsan, Chris - ok

File: test/Dialect/XTenNN/simplify.mlir (hunk 20 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/simplify.mlir:1]: Max -ok

File: test/Dialect/XTenNN/subgraph-invalid.mlir (hunk 21 above)
Comment (laurettaSchubert) [reply test/Dialect/XTenNN/subgraph-invalid.mlir:1]: Max - ok

File: test/Transform/XTenMinimizeLiveTensors/diamonds.mlir (hunk 22 above)
Comment (laurettaSchubert) [reply test/Transform/XTenMinimizeLiveTensors/diamonds.mlir:1]: Tiago - ok

File: test/Transform/XTenMinimizeLiveTensors/failures.mlir (hunk 23 above)
Comment (laurettaSchubert) [reply test/Transform/XTenMinimizeLiveTensors/failures.mlir:1]: Tiago - ok

File: test/Transform/XTenMinimizeLiveTensors/other_subgraphs.mlir (hunk 24 above)
Comment (laurettaSchubert) [reply test/Transform/XTenMinimizeLiveTensors/other_subgraphs.mlir:1]: Ferdinand, Tiago -ok

File: test/Transform/XTenMinimizeLiveTensors/print-dot-graph.mlir (hunk 25 above)
Comment (laurettaSchubert) [reply test/Transform/XTenMinimizeLiveTensors/print-dot-graph.mlir:1]: Tiago - ok

File: test/aten-opt/commandline.mlir (hunk 26 above)
Comment (laurettaSchubert) [reply test/aten-opt/commandline.mlir:1]: Tina, Max -ok

File: test/aten-opt/outputfile.mlir (hunk 27 above)
Comment (laurettaSchubert) [reply test/aten-opt/outputfile.mlir:1]: Max - ok

**Comment by laurettaSchubert** [comment]
All good, but looking at the file paths we might want to exclude the 'test' folder from the list and maybe also the 'utils' folder in general. Adding @ljfitz as reviewer now to give some feedback.
@Ishirui please store the config files for generating this in the ticket - thanks

File: lib/Conversion/XTenNNToTorch.cpp (hunk 3 above)
Comment (laurettaSchubert) [reply lib/Conversion/XTenNNToTorch.cpp:1]: Liam -ok

File: test/Conversion/XTenNNToLinalg/Elu.mlir (hunk 5 above)
Comment (laurettaSchubert) [reply test/Conversion/XTenNNToLinalg/Elu.mlir:1]: Tiago, Robert - ok

**Review by laurettaSchubert — APPROVED** [review]
Looking all good

**Review by ljfitz — APPROVED** [review]

**Pull request merged**